engine/polyglot-api/src/main/scala/org/enso/polyglot/runtime/Runtime.scala

```diff
@@ -385,10 +385,22 @@ object Runtime {
       sealed trait Payload
       object Payload {
 
-        /** An empty payload. Indicates that the expression was computed to a
-          * value.
+        /** Indicates that the expression was computed to a value.
+          *
+          * @param warnings information about attached warnings.
           */
-        case class Value() extends Payload
+        case class Value(warnings: Option[Value.Warnings] = None)
```

Comment:
This is pretty annoying that we have this duplication but I guess there is not much we can do.

Reply:
We probably cannot start new project, but we could make incremental steps:
- share `case class` between both APIs step by step
- whenever we introduce new type, make sure the it is shared

Would that work?